Resources/public/js/ajax-updateOnChange.js

```diff
@@ -0,0 +1,36 @@
+$(document).on(
+    'change',
+    'select[data-refreshOnChange="true"], ' +
+    'input:checkbox[data-refreshOnChange="true"], ' +
+    'input:radio[data-refreshOnChange="true"]',
+    function(event) {
+        var form = $(this).parents('form');
+        var formdata = (window.FormData) ? new FormData(form[0]) : null;
+        var data = (formdata !== null) ? formdata : form.serialize();
+        var ignoredChanges = {};
+        $(form).find('[data-ignoreonchange]').each(function (idx, elem) {
+            var $elem = $(elem);
+            ignoredChanges[$elem.attr('data-ignoreonchange')] = $elem;
+        });
+        $.ajax({
+            url         : form.attr('action') + (form.attr('action').split('?')[1] ? '&' : '?') + 'novalidate',
+            data        : data,
+            type        : $(form).attr('method'),
+            contentType : false,
+            dataType    : 'json',
+            processData : false,
+        }).done(function(response){
+            if (typeof response === 'object' && response.hasOwnProperty("html")) {
+                form.html(response.html);
+            } else {
+                form.html(response);
+            }
+            $.each(ignoredChanges, function(index, elem){
+                $('[data-ignoreonchange="' + index + '"]').replaceWith(elem);
+            });
+        }).fail(function(response) {
+            console.log(response);
+            error('Oups, une erreur est apparue', 10000);
```

Comment:
Don't put a french sentence.
An elegant way would to use a translation files or better, a translator like BazingaJsTranslationBundle.
At least, if you don't have so much time, please use english.